src/main/java/org/radarcns/mongo/util/MongoHelper.java

```diff
@@ -57,29 +57,30 @@
     public static final int DESCENDING = -1;
 
     /**
-     * Enumerate all available statistical values. The string value represents the Document field
-     * that has to be used to compute the result.
+     * Finds whether document is available for given query parameters.
+     * https://stackoverflow.com/a/8390458/822964 suggests find().limit(1).count(true) is the
+     * optimal way to do it.
+     *
+     * @param projectName of the project
+     * @param subjectId is the subjectID
+     * @param sourceId is the sourceID
+     * @param start is the start time of the queried timewindow
+     * @param end is the end time of the queried timewindow
+     * @param collection is the MongoDB that will be queried
+     * @return a MongoDB cursor containing all documents from the query.
      */
-    public enum Stat {
-        avg("avg"),
-        count("count"),
-        iqr("iqr"),
-        max("max"),
-        median("quartile"),
-        min("min"),
-        quartile("quartile"),
-        receivedMessage("count"),
-        sum("sum");
-
-        private final String param;
-
-        Stat(String param) {
-            this.param = param;
-        }
+    public static MongoCursor<Document> doesExistsByProjectAndSubjectAndSourceInWindowForStartTime(
```

Comment:
Perhaps this can be just `doesSourceExist` and document that the data is specified for only a project, subject and time window.

Reply:
How about `findFirstDocumentForStartTimeWindow`. It essentially finds the first document that matches the query, something like whether we have any records or not.